README.md

```diff
@@ -19,7 +19,10 @@ up the EKS Distro from source.
 
 ## Releases
 
-Full documentation for releases can be found on https://distro.eks.amazonaws.com
+Full documentation for releases can be found on [https://distro.eks.amazonaws.com](https://distro.eks.amazonaws.com).
```

Comment:
This is silly, but GitHub included the period in the URL when I added it to the end of this sentence. So this change fixes this issue